Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -607,10 +568,35 @@ function (FixerInterface $a, FixerInterface $b) {
             }
         }
 
+        $rulesSets = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
```

Comment:
I hate avoiding statics by using weird workarounds ;)

If you do not want a static method here, you should move this part to another (maybe new) class in my opinion.